miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java

```diff
@@ -380,25 +381,28 @@ private void applyChanges(Run target, Run source) throws IOException {
     if (isIlluminaRun(target)) applyIlluminaChanges((IlluminaRun) target, (IlluminaRun) source);
     if (isPacBioRun(target)) applyPacBioChanges((PacBioRun) target, (PacBioRun) source);
     if (isLS454Run(target)) applyLS454Changes((LS454Run) target, (LS454Run) source);
+    if (isSolidRun(target)) applySolidChanges((SolidRun) target, (SolidRun) source);
```

Comment:
This and above can be 'else ifs'

Remove `target.setPairedEnd` above since it should only be set on specific subclasses